lib/identity_doc_auth/lexis_nexis/error_generator.rb

```diff
@@ -114,24 +102,24 @@ def get_error_messages(liveness_enabled, response_info)
             alert_msg_hash = TRUE_ID_MESSAGES[alert[:name].to_sym]
 
             if alert_msg_hash.present?
-              errors[alert_msg_hash[:type]].add(config.i18n.t(alert_msg_hash[:msg_key]))
+              errors[alert_msg_hash[:type]] << alert_msg_hash[:msg_key]
```

Comment:
just confirming, but `errors` is a hash whose values are sets, so this is adding an error to that set?

Reply:
Yeah, but then they get converted to an array:

https://github.com/18F/identity-doc-auth/blob/f0b2813e5f0fadefdb1a6ce1edd589a812893886/lib/identity_doc_auth/lexis_nexis/error_generator.rb#L103